config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/shelters', to: 'shelters#index'
+  get '/shelters/new', to: 'shelters#new'
```

Comment:
Note to self: #new didn't work until it was moved about #show because rails looks for the first thing it matches